src/style.css

```diff
@@ -0,0 +1,82 @@
+@font-face {
+  font-family: Avenir;
+  src: url("./fonts/Avenir.ttc");
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: bold;
+  src: url("./fonts/Avenir-Heavy.ttf");
+}
```

Comment:
how is the code supposed to distinguish between these two fonts if they have the same name?